src/Sulu/Bundle/MediaBundle/Api/Media.php

```diff
@@ -358,6 +358,17 @@ public function getVersion()
     }
 
     /**
+     * @VirtualProperty
+     * @SerializedName("subVersion")
+     *
+     * @return int
+     */
+    public function getSubVersion()
+    {
+        return $this->getFileVersion()->getSubVersion();
```

Comment:
What's the subversion for? Why not increment the normal file version instead? If we start to introduce a different version we'll adding a lot of complexity.